main.go

```diff
@@ -105,7 +110,16 @@ func main() {
 		if len(includeSecretsDataGlobs) == 0 {
 			includeSecretsDataGlobs = args.GlobArgs([]string{"*"})
 		}
-		configChecker := checkers.NewSecretChecker(pollingPeriod, secretsLabelSelector, includeSecretsDataGlobs, excludeSecretsDataGlobs, secretsAnnotationSelector, secretsNamespace, kubeconfigPath, &exporters.SecretExporter{}, includeSecretsTypes)
+		var secretsNamespaces []string
+		if len(secretsListOfNamespaces) > 0 {
```

Comment:
let's make `getSanitizedNamespaceList()` take both parameters and consolidate the logic of building a slice there. it will clean up the logic here and below. something like:

`allNamespaces := getSanitizedNamespaceList(secretsListOfNamespace, secretsNamespace)`

Reply:
Done